bookwyrm/tests/test_book_search.py

```diff
@@ -134,3 +135,188 @@ def parse_isbn_search_data(self, data):
         # there's really not much to test here, it's just a dataclass
         self.assertEqual(result.confidence, 1)
         self.assertEqual(result.title, "Title")
+
+
+class SearchVectorTest(TestCase):
+    """check search_vector is computed correctly"""
```

Comment:
Uhm I'm not sure how, but I managed to completely miss tests/test_progress.py.

I think everything tested there is tested here, though triggers are tested functionally, rather than structurally (i.e. for trigger behavior we look at search results, not at search_vector). I can bring those missing tests from there to here, and have it all in this file.

In any case I apologize for the confusion; it wasn't intentional. :)

Reply:
I think consolidating them here is best

Reply:
Will do.